python/py-onnx/Portfile

```diff
@@ -4,40 +4,39 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-onnx
-version             1.14.1
-
-checksums           md5 2ae765ea11027f44228b4f012b256cd3 \
-                    rmd160 189124ae50eb0f3a8f3aaeeb51497a3cce8e96ef \
-                    sha256 70903afe163643bd71195c78cedcc3f4fa05a2af651fd950ef3acbb15175b2d1
-
+version             1.16.1
 categories-append   science
-
 license             Apache-2
-
 maintainers         nomaintainer
 
 description         Open Neural Network Exchange
 
-long_description    {*}${description} (ONNX) is an open\
-                    ecosystem that empowers AI developers to choose\
-                    the right tools as their project evolves. ONNX\
-                    provides an open source format for AI models, both\
-                    deep learning and traditional ML. It defines an\
-                    extensible computation graph model, as well as\
-                    definitions of built-in operators and standard\
-                    data types. Currently we focus on the capabilities\
+long_description    {*}${description} (ONNX) is an open \
+                    ecosystem that empowers AI developers to choose \
+                    the right tools as their project evolves. ONNX \
+                    provides an open source format for AI models, both \
+                    deep learning and traditional ML. It defines an \
+                    extensible computation graph model, as well as \
+                    definitions of built-in operators and standard \
+                    data types. Currently we focus on the capabilities \
                     needed for inferencing (scoring).
 
 homepage            https://onnx.ai/
 
-python.versions     38 39 310 311
+checksums           md5     07fe47dc50e802e4a9051de9b3cdf928 \
+                    rmd160  e9817f10bbce05a2a7d5a716c7557467637c5287 \
+                    sha256  8299193f0f2a3849bfc069641aa8e4f93696602da8d165632af8ee48ec7556b6 \
+                    size    12306956
+
+python.versions     38 39 310 311 312
 
 if {${name} ne ${subport}} {
 
     compiler.cxx_standard 2011
 
     depends_build-append \
         path:bin/cmake:cmake \
+        port:py${python.version}-cmake \
```

Comment:
I don't see why this would be necessary - what's the problem with using the already existing `cmake` port?

Reply:
Without this dependency, this build error is thrown:

```sh
:info:build writing manifest file 'onnx.egg-info/SOURCES.txt'
:info:build ERROR Missing dependencies:
:info:build     cmake
:info:build Command failed:  cd "/opt/local/var/macports/build/_opt_local_ports_python_py-onnx/py312-onnx/work/onnx-1.16.1" && /opt/local/Library/Frameworks/Python.framework/Versions/3.12/bin/python3.12 -m build --no-isolation --wheel --outdir /opt/local/var/macports/build/_opt_local_ports_python_py-onnx/py312-onnx/work 
:info:build Exit code: 1
```

Reply:
Okay, thank you. I've done that, and it works.